docs/guides/run-primitives-rest-api.mdx

```diff
@@ -0,0 +1,489 @@
+---
+title: Run primitives with REST API
+description: How to use the Sampler and Estimator primitives with Qiskit Runtime REST API.
+---
+
+# Run primitives with REST API
+
+The steps in this topic describe how to run and configure primitive workloads with REST API, and demonstrate how to invoke them in any program of your choice.
+
+<Admonition type="note">
+    This documentation utilizes the Python `requests` module to demonstrate the Qiskit Runtime REST API. However, this workflow can be executed using any language or framework that supports working with REST APIs. Refer to the [API reference documentation](/api/runtime/tags/jobs) for details.
+</Admonition>
+
+<span id="start-estimator"></span>
+## Estimator primitive with REST API
+
+### 1. Initialize the account
+
+Because Qiskit Runtime `Estimator` is a managed service, you first need to initialize your account. You can then select the compute resource you want to use to calculate the expectation value. 
```

Comment:
There had been 3 sentences here but the last two seemed very similar, so I removed one. If I've affected the meaning please do put it back :)